install.sh

```diff
@@ -19,6 +19,9 @@ trap_with_arg cleanup ERR INT TERM EXIT
 source install/check-latest-commit.sh
 source install/check-minimum-requirements.sh
 
+echo "BAILING OUT EARLY"
```

Comment:
This does not look like this should be here 🤔

Reply:
🤦 thank you; my `git commit -a` mistake